Utils/Dataflow/055_documents2TTL/documents2ttl.py

```diff
@@ -460,21 +533,25 @@ def fix_string(wrong_string):
 
 
 def fix_list_values(list_vals):
-    """
-    Fixing list values with fix_string
-    :param list_vals:
-    :return:
+    """ Apply fix_string to each item in a list.
+
+    :param list_vals: list with strings to be fixed
+    :type list_vals: list
+
+    :return: list with fixed strings
+    :rtype: list
     """
     list_vals = [fix_string(item) for item in list_vals]
     return list_vals
 
 
 def process(stage, msg):
-    """
-    Processing messages from JSON to TTL
-    :param stage: instance of JSON2TTLProcessorStage
-    :param msg: input JSON message
-    :return:
+    """ Process a message. Convert the message's contents from JSON to TTL.
```

Comment:
432bb25
Added link to original `process()` and mentioned the output message.

While "process a message" is indeed not very informative, I think it should be left as is because its the shortest way to describe the function, and that's the goal of this single line. More detailed information such as "what is in the message" or "where do the results go" can be obtained two lines lower.

Reply:
Done in dd69a67.